docs/resources/gateway_plugin_ai_mcp_proxy.md

```diff
@@ -45,10 +45,11 @@ resource "konnect_gateway_plugin_ai_mcp_proxy" "my_gatewaypluginaimcpproxy" {
         method = "GET"
         parameters = [
           {
-            description = "...my_description..."
-            in          = "...my_in..."
-            name        = "...my_name..."
-            required    = true
+            additional_properties = "{ \"see\": \"documentation\" }"
```

Comment:
Does this schema have a field named `additional_properties`? It doesn't feel right to me

Reply:
Missed this one - yes, it is not supposed to be there.
Based on jsonschema, this is a free form object, but we have an overlay in place which declares the keys.